README.md

```diff
@@ -1 +1,31 @@
 # bedrock-ledger-test-cli
+
+Setup:
```

Comment:
This should be a heading ## or ###

Reply:
Addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd

Reply:
My comment was in reference to L3 'Setup:' and the patch changes the number of pound signs on L1.